test/spec/features/popup-menu/PopupMenuComponentSpec.js

```diff
@@ -6,9 +6,7 @@ import {
 } from 'lib/ui';
 
 import {
-  bootstrapDiagram,
-  insertCSS,
-  inject
+  insertCSS
```

Comment:
Please add tests that verify the API in the API spec (`PopupMenuSpec`). This component may be dead, but the contract we setup shall remain.